jekyll/_cci2/scheduled-pipelines.md

```diff
@@ -27,10 +27,10 @@ Since the scheduled run is based on pipelines, scheduled pipelines have all the
 
 - Control the actor associated with the pipeline, which can enable the use of [restricted contexts]({{site.baseurl}}/2.0/contexts/#restricting-a-context).
 - Use [dynamic config]({{site.baseurl}}/2.0/dynamic-config/) via setup workflows.
-- Modify the schedule without having to edit `config.yml`.
-- Interact with auto-cancelling of pipelines.
-- Specify pipeline parameters associated with a schedule.
-- Consolidate the management of common schedules.
+- Modify the schedule without having to edit `.circleci/config.yml`.
+- Take advantage of auto-cancelling.
+- Specify [pipeline parameters]({{site.baseurl}}/2.0/pipeline-variables/#pipeline-parameters-in-configuration) associated with a schedule.
+- Manage common schedules, e.g. across workflows.
```

Comment:
Sounds like a good use case for an example for the config cookbook maybe. I'll make a ticket :-)

Reply:
Great.

Reply:
Hola Rosie, Gracias.